mantle/platform/cluster.go

```diff
@@ -180,14 +180,7 @@ func (bc *BaseCluster) Keys() ([]*agent.Key, error) {
 }
 
 func (bc *BaseCluster) RenderUserData(userdata *platformConf.UserData, ignitionVars map[string]string) (*platformConf.Conf, error) {
-	if userdata == nil {
-		switch bc.IgnitionVersion() {
-		case "v3":
-			userdata = platformConf.Ignition(`{"ignition": {"version": "3.0.0"}}`)
-		default:
-			return nil, fmt.Errorf("unknown ignition version")
-		}
-	}
+	userdata = platformConf.Ignition(`{"ignition": {"version": "3.0.0"}}`)
```

Comment:
This now unconditionally overrides the passed-in userdata.

Reply:
Oh, nice catch again! Thanks!